pages/sales-pos/index.vue

```diff
@@ -113,60 +239,75 @@ import { useNftStore } from '~/stores/nft'
 import { useCollectionStore } from '~/stores/collection'
 
 const route = useRoute()
+const toast = useToast()
 const { LIKER_LAND_URL } = useRuntimeConfig().public
 const nftStore = useNftStore()
 const collectionStore = useCollectionStore()
 
-const { lazyFetchClassMetadataById } = nftStore
+const { lazyFetchClassMetadataById, lazyFetchClassListingInfoById } = nftStore
 const { lazyFetchCollectionById } = collectionStore
 
-const newProductId = ref('')
+const newProductIdInputs = ref<{ id: number; value: string, error?: Error }[]>([{ id: 0, value: '' }])
+const newProductIdInputNextId = ref(1)
 const giftToEmail = ref('')
 const saleItemList = ref<any[]>([])
+const isShowAddItemModal = ref(false)
 const isOpenQRCodeModal = ref(false)
 const isEditMode = ref(false)
 
+watch(isShowAddItemModal, (value) => {
+  if (!value) {
+    newProductIdInputs.value = [{ id: 0, value: '' }]
+    newProductIdInputNextId.value = 1
+  }
+})
+
 const saleItemTableColumns = computed(() => {
   const columns = [
-    { key: 'selected', label: 'Selected' },
-    { key: 'image', label: 'Cover' },
-    { key: 'name', label: 'Name' },
-    { key: 'priceIndex', label: 'Edition' }
+    { key: 'name', label: 'Product', class: '' }
   ]
   if (isEditMode.value) {
-    columns.push({ key: 'delete-action', label: 'Delete' })
+    columns.push({ key: 'delete-action', label: '', class: 'w-12' })
   }
   return columns
 })
 
-const saleItemTableRows = computed(() => {
-  return saleItemList.value.map((item, index) => {
-    return item.collectionId
-      ? {
-          collectionId: item.collectionId,
-          priceIndex: undefined,
-          selected: { value: item.selected, class: item.selected ? 'bg-green-500/50 animate-pulse' : '' },
```

Comment:
Is there a way to make selected row more obvious? Now it is just a slightly darker grey, quite hard to see
<img width="1011" alt="image" src="https://github.com/user-attachments/assets/d0d51ef9-d0b6-4042-b30f-7b785e470fa2">